hw/ip/edn/doc/_index.md

```diff
@@ -19,7 +19,9 @@ A primary objective of the EDN block is to provide a simpler hardware interface
 - There are two interrupts that are supported:
   - CSRNG application command has completed.
   - An internal FIFO error has occurred.
-- No alerts are supported by this revision.
+- There are two alerts that are implemented in is revision:
+  - A fatal alert to report common security fatal errors and EDN specific fatal errors. A list of fatal errors are listed in the {{< regref "ERR_CODE" >}} register.
+  - A recoverable alert to report recoverable error. A list of fatal errors are listed in the {{< regref "RECOV_ALERT_STS" >}} register.
```

Comment:
I just want to suggest these two minor updates while reading the doc :) 
Please feel free to create a different PR if you prefer some better wordings.